### PR TITLE
Workaround for stale pingsource resources

### PR DESCRIPTION
**Pull request by mgencur**

### CODE REVIEW

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

File: hack/lib/serverless.bash [hunk 1]
```diff
@@ -169,6 +169,8 @@ function teardown_serverless {
   if oc get knativeeventing.operator.knative.dev knative-eventing -n "${EVENTING_NAMESPACE}" >/dev/null 2>&1; then
     logger.info 'Removing KnativeEventing CR'
     oc delete knativeeventing.operator.knative.dev knative-eventing -n "${EVENTING_NAMESPACE}" || return $?
+    # TODO: Remove workaround for stale pingsource resources (https://issues.redhat.com/browse/SRVKE-473)
+    oc delete deployment -n "${EVENTING_NAMESPACE}" --ignore-not-found=true pingsource-mt-adapter
```
Comment (matzew) [inline hack/lib/serverless.bash:173]: /lgtm
/approve

**Comment by matzew** [comment]
```
luster operator authentication Degraded is True with IngressStateEndpoints_MissingSubsets::OAuthServiceCheckEndpointAccessibleController_SyncError::OAuthServiceEndpointsCheckEndpointAccessibleController_SyncError: OAuthServiceEndpointsCheckEndpointAccessibleControllerDegraded: oauth service endpoints are not ready\nOAuthServiceCheckEndpointAccessibleControllerDegraded: Get \"https://172.30.191.61:443/healthz\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)\nIngressStateEndpointsDegraded: No subsets found for the endpoints of oauth-server"
level=info msg="Cluster operator authentication Progressing is Unknown with NoData: "
```


/test 4.6-upstream-e2e-aws-ocp-46

**Pull request merged**